compose/ui/ui/src/nonJvmMain/kotlin/androidx/compose/ui/node/SortedSet.nonJvm.kt

```diff
@@ -112,6 +112,12 @@ internal actual class SortedSet<E> actual constructor(
             }
         }
 
+        // FIXME: https://youtrack.jetbrains.com/issue/KT-82783
+        if (indexByElement.size != itemTree.size) {
+            indexByElement.clear()
+            itemTree.forEachIndexed { index, element -> indexByElement[element] = index }
```

Comment:
We change the complexity of `remove()` from `O(log n)` to `O(n)`. It might affect performance.

- If there is no other option, we need to run benchmarks. I am not sure we have one that measures this code, so probably we need to write adding/removing some code in the composition, and measure FPS?
- can we wait for the hotfix in Kotlin instead? It might not a new issue and we live with it for awhile?